packages/client/bin/startRpc.ts

```diff
@@ -176,7 +176,7 @@ export function startRPCServers(client: EthereumClient, args: RPCArgs) {
       rpcDebug,
       rpcDebugVerbose,
       logger,
-    })
+    }) //@ts-ignore
```

Comment:
Why is this necessary? Can we just typecast instead?

Reply:
I think this was an earlier left over.  Looks like we don't need it at all.  Will remove.